packages/wdio-browserstack-service/src/accessibility-handler.ts

```diff
@@ -0,0 +1,343 @@
+import logger from '@wdio/logger'
+import type { Capabilities, Frameworks } from '@wdio/types'
+
+import type { ITestCaseHookParameter } from './cucumber-types.js'
+
+import {
+    getA11yResultsSummary,
+    getA11yResults,
+    getUniqueIdentifier,
+    getUniqueIdentifierForCucumber,
+    isAccessibilityAutomationSession,
+    isBrowserstackSession,
+    o11yClassErrorHandler,
+    shouldScanTestForAccessibility,
+    validateCapsWithA11y,
+    isTrue,
+} from './util.js'
+import type { BrowserStack } from './types.js'
+import type { BrowserStackCapabilities } from '@wdio/types/build/Capabilities.js'
```

Comment:
never import from files directly because authors might move around these interfaces. You already imported the package at the top of the file.

Use the type via `Capabilities.BrowserStackCapabilities`
```suggestion
```